config_app/config_endpoints/setup_web.py

```diff
@@ -8,6 +10,27 @@
 setup_web = Blueprint("setup_web", __name__, template_folder="templates")
 
 
+def _get_readonly_fields():
+    """
+    Returns a list of fields which should be marked as "Read Only" in the
+    UI as they are most likely managed by an external service such as an
+    Operator.
+
+    This expects an environment variable with the following format:
+    READ_ONLY_FIELDS=redis,hostname
+    """
+    environment_variable_name = "CONFIG_READ_ONLY_FIELDS"
+    options = ("redis", "hostname")
```

Comment:
I'd prefer if we just passed it through a very basic regex (like `[a-z]{1,30}` or somesuch)

Reply:
Sounds good! I removed the `options` and used the same regex you suggested